wdfn-server/waterdata/views.py

```diff
@@ -215,7 +215,7 @@ def networks(network_cd):
     if network_cd:
         collection = network_data
         extent = network_data['extent']['spatial']['bbox'][0]
-        narrative = markdown(network_data['properties']['narrative']) if network_data['properties']['narrative'] else None
+        narrative = markdown(network_data['properties']['narrative']) if network_data['properties'].get('narrative') else None
```

Comment:
You could more tersely write:
```
narrative = markdown(network_data['properties'].get('narrative')
```
See: https://www.w3schools.com/python/ref_dictionary_get.asp

Reply:
Does `markdown` barf on `None`?  I assumed that was why it was structured that way.

Reply:
lol. I just asked Mary about that. I think that is the case.

Reply:
Okay, seems like Markdown is the issue, so I guess I'll leave it as is.